README.md

```diff
@@ -1,45 +1,75 @@
-# Catapult-explorer
+# catapult-explorer
```

Comment:
Most Catapult projects and libraries prepend "nem2". Should we rename ``catapult-explorer`` to ``nem2-explorer``?

Reply:
@evias what you think?
i'm ok with nem2-explorer